kaldb/src/main/java/com/slack/kaldb/writer/kafka/KaldbKafkaConsumer.java

```diff
@@ -38,57 +41,81 @@ public class KaldbKafkaConsumer {
   public static final int KAFKA_POLL_TIMEOUT_MS = 250;
   private final LogMessageWriterImpl logMessageWriterImpl;
 
+  private static final String[] REQUIRED_CONFIGS = {ConsumerConfig.BOOTSTRAP_SERVERS_CONFIG};
+  private static final Set<String> OVERRIDABLE_CONFIGS =
+      Set.of(
+          ConsumerConfig.KEY_DESERIALIZER_CLASS_CONFIG,
+          ConsumerConfig.VALUE_DESERIALIZER_CLASS_CONFIG,
+          ConsumerConfig.AUTO_OFFSET_RESET_CONFIG);
+
   public static KaldbKafkaConsumer fromConfig(
```

Comment:
Is this method still useful now that it just passes its three parameters to a public constructor with the same three parameters?

Reply:
good catch! removed it